lib/device_api/android/device.rb

```diff
@@ -253,6 +253,13 @@ def diskstat
       def uptime
         ADB.get_uptime(serial)
       end
+
+      # Returns the Wifi mac address
+      def wifi_mac_address
+        network = ADB.get_network_info(serial)
+        wifi = network.find { |a| a[:name] == 'wlan0' }
```

Comment:
Prefer `detect` over `find`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.